performance_regression_test.py

```diff
@@ -252,6 +266,8 @@ def run_read_workload(self, nemesis=False):
             self.db_cluster.add_nemesis(nemesis=self.get_nemesis_class(), tester_obj=self)
             self.db_cluster.start_nemesis(interval=interval)
         results = self.get_stress_results(queue=stress_queue)
+
+        self.build_histogram(PerformanceTestWorkload.READ, PerformanceTestType.LATENCY)
```

Comment:
Shouldn't it be used only when `use_hdr_cs_histogram` is enabled?
The same for all other usages of HDR feature.

Reply:
Added check that flag is enabled